net-p2p/qbittorrent/qbittorrent-5.0.5.ebuild

```diff
@@ -33,9 +33,9 @@ REQUIRED_USE="
 
 RDEPEND="
 	>=dev-libs/openssl-3.0.2:=
-	>=net-libs/libtorrent-rasterbar-1.2.19:=
-	>=sys-libs/zlib-1.2.11
 	>=dev-qt/qtbase-6.5:6[network,ssl,sql,sqlite,xml]
+	>=net-libs/libtorrent-rasterbar-2.0.10:=
```

Comment:
I think this deserves more comment in the commit message with regard to why we are changing the libtorrent-rasterbar version. "due to the drop of :1" doesn't tell me why it was ok to use such a low version to begin with.

The background for this -- which I do happen to know -- is that upstream supports a certain minimum version of 1.x and a certain minimum version of 2.x, and this ebuild was trying to accommodate the former. Since a `|| ( ... )` dependency cannot bind the slot to `:=` we weren't actually able to enforce a minimum 2.x version correctly, which was bad. But historically, the package added at least the minimum bounds for the 1.x version because that's what a lot of people were sticking with due to teething issues in newer libtorrent-rasterbar.

Reply:
Is it really obvious that one can now do >=2 instead of >=1 when nothing changed upstream? No.

Reply:
Yes. We should try (if possible / straightforward) to write commit messages that help other people who don't know the package to understand what the commit is doing. :) This helps e.g. for cases where people are reviewing recent changes to many packages.